test/integration/contract-aci.js

```diff
@@ -301,6 +301,20 @@ describe('Contract instance', function () {
 
       expect(testContract.decodeEvents(log)).to.be.eql(eventResult.decodedEvents)
     })
+
+    it('calls a contract without events that calls another contract emitting events', async () => {
+      const contract = await sdk.getContractInstance({
+        source:
+          'contract interface StateContract =\n' +
+          '  stateful entrypoint emitEvents: () => unit\n' +
```

Comment:
@davidyuk not sure if this method is emitting any events? What does `unit` stands for? Looked into the Sophia documentation, Did not find anything on it.

Reply:
Got it. L308 contract is just an interface for instance creation. Thank you.